CHANGELOG.txt

```diff
@@ -1,3 +1,514 @@
+0a969db2b Bugfix: Plugins endpoint was unauthenticated (#14570)
+630aeff72 Fix AzureDataFactoryHook failing to instantiate its connection (#14565)
+22e3a4cc0 BugFix: fix DAG doc display (especially for TaskFlow DAGs) (#14564)
+45a0ac2e0 BugFix: TypeError in monitor_pod (#14513)
+0aa597e2f Fix statsd metrics not sending when using daemon mode (#14454)
+59c459fa2 BugFix: Serialize max_retry_delay as a timedelta (#14436)
+fe0ee585d Fix comparison dagTZ with localTZ (#14204)
+8f21fb1bf fix lossing duration < 1 secs in tree (#13537)
+3a046faae Remove testfixtures module that is only used once (#14318)
+21f297425 Fix crash when user clicks on  "Task Instance Details" caused by start_date being None (#14416)
+258ec5d95 Fix logging error with task error when JSON logging is enabled (#14456)
+6c6b77a87 Fix permission error on non-POSIX filesystem (#13121)
+8c5e61823 Fix spelling (#14472)
+a48bedf26 Fix spelling in "ignorable" (#14348)
+79ffe03a6 Fix spelling (#14343)
+7d82a046b Fix get_context_data doctest import (#14288)
+e3bcaa3ba Correct typo in GCSObjectsWtihPrefixExistenceSensor  (#14179)
+fc67521f3 Fix order of failed deps (#14036)
+eb78a8b86 Bugfix: Scheduler fails if task is removed at runtime (#14057)
+604a37eee Fix broken SLA Mechanism (#14056)
+b59e416f6 Fix critical CeleryKubernetesExecutor bug (#13247)
+629abfdba Bugfix: Fix permissions to triggering only specific DAGs  (#13922)
+833e33832 Fix four bugs in StackdriverTaskHandler (#13784)
+38f085396 Fix static check on Master (#13721)
+4be27af04 Fix the behavior for deactivate the authentication option and documenting the process to do it (#13191)
+053d90381 `func.sum` may returns `Decimal`  that break rest APIs (#15585)
+f878ec6c5 Persist tags params in pagination (#15411)
+d89bcad26 API: Raise `AlreadyExists` exception when the execution_date is same (#15174)
+116a8a0c6 Remove duplicate call to sync_metadata inside DagFileProcessorManager (#15121)
+b0e68ebcb Avoid scheduler/parser manager deadlock by using non-blocking IO (#15112)
+41930fdeb extra docker-py update to resolve docker op issues (#15731)
+35b5a3831 Don't add User role perms to custom roles. (#13856)
+756031608 Import Connection lazily in hooks to avoid cycles (#15361)
+16902d043 Ensure executors end method is called (#14085)
+7087541a5 Remove user_id from API schema (#15117)
+ca4c4f3d3 Re-introduce dagrun.schedule_delay metric (#15105)
+375d26d88 Pass queue to BaseExecutor.execute_async like in airflow 1.10 (#14861)
+78d9feb5f Removes references to directory that no longer exists (#15083)
+57388ef65 Prevent clickable bad links on disabled pagination (#15074)
+def961512 Scheduler: Remove TIs from starved pools from the critical path. (#14476)
+6d3046431 Compare string values, not if strings are the same object (#14942)
+5d96eb0e0 Acquire lock on db for the time of migration (#10151)
+a85d84020 Make pytest collection warnings errors (#14832)
+303c89fea Return output of last task from task_group wrapper. (#15779)
```

Comment:
```suggestion
```

This is a change to a new feature before it was released, so it doesn't make sense to have both in the changelog